internal/pkg/helm/values_inspector.go

```diff
@@ -91,3 +100,141 @@ func isBooleanYamlValue(input *map[string]interface{}, path *[]string, pathIndex
 		}
 	}
 }
+
+//GetValues .. Get values  fro given path
+func GetValues(arg string) (*map[string][]*LogHelmReport, error) {
+	argString := strings.ReplaceAll(arg, ".Values.", "")
+	chartClient := NewChartClient()
+	err := chartClient.LoadChartFrom(HelmChartRef)
+	if err != nil {
+		logrus.Warnf("error loading chart: %s", err)
+		return nil, err
+	}
+	raw, _ := chartutil.ReadValues([]byte(chartClient.Chart.Values.Raw))
+	result, err := raw.PathValue(argString)
+	if err != nil {
+		logrus.Warnf("Path value not found for path : %s ,", argString)
```

Comment:
What is the trailing comma for at the end of this string?

Reply:
Good catch ,  I will clean it up